### ISSUE TITLE
obstacle 

### ISSUE THREAD

**Issue by antoine-tesson** [cnigfr/schema-accessibilite-voirie#50]
Une piste pour gagner du temps sur la récolte

Les attributs longueur et largeur de l'obstacle, bien que facultatifs, nous semblent parfaitement inutiles tant dans l'optique d'un calculateur que pour un PAVE. Il est par ailleurs très difficile de mesurer la largeur précise d'un arbre ou de certains mobiliers urbains quand la largeur n'est pas uniforme sur toute la hauteur.
Je propose de les supprimer du standard. "Mais ce n'est pas gênant de les laisser puisqu'ils sont facultatifs me direz vous", pas faux mais je crains que certaines collectivité ne demande des relevés de tout les attributs et franchement c'est une horreur à saisir. On perd du temps pour rien puisque le calculateur d'itinéraire n'a besoin que de la largeur utile (UFR) et hauteur sous obstacle (déficients visuels).
D'ailleurs, je propose de rendre ce dernier attribut obligatoire car il s'agit d'un danger de heurt à la tête.

**Comment by GT-CNIG-DDU** [comment]
Je n'ai pas d'avis bien tranché sur l'utilité des attributs `OBSTACLE.largeurObstacle` et `OBSTACLE.longueurObstacle`, mais les réunions du SG ERDA nous donneront potentiellement des éléments d'appréciations, en particulier sur l'utilité et le statut de `OBSTACLE.hauteurSousObs`

Intuitivement `OBSTACLE.longueurObstacle` me semble effectivement plus utile que `OBSTACLE.largeurObstacle` (du fait de la présence de l'attribut obligatoire `OBSTACLE.largeurUtile`)

D'ailleurs les deux attributs n'ont pas le même statut : le premier est obligatoire à remplissage facultatif, le deuxième est purement optionnel, donc à la discrétion de la collectivité maître d'ouvrage de la collecte.

Discussion à rapprocher sans doute de #44.

**Comment by Marion-Torterotot** [comment]
Hello,

OK pour moi pour supprimer l'attribut `OBSTACLE.largeurObstacle` , même si je pense que c'est peut-être un faux sujet car on pourrait envisager de le remplir automatiquement à partir de la formule : `CIRCULATION.largeurUtile` - `OBSTACLE.largeurUtile`

Pour l'attribut `OBSTACLE.longueurObstacle` , je suis plus partagée : ça peut donner une info supplémentaire pour le calculateur (à voir si c'est "intégrable") : un obstacle court (voire très court) est plus facile à contourner qu'un obstacle long. Ca permet aussi d'éviter de se poser la question de créer un autre tronçon pour prendre en compte une largeur utile plus petite sur une "longue" distance. Je reconnais que c'est un peu fastidieux à relever, mais quand tu as plusieurs fois le même obstacle (typiquement une suite de candélabres), tu ne fais la mesure qu'une fois.
Par ailleurs, j'ai cru comprendre que c'était aussi un moyen de compenser le fait que NeTEx ne prévoit pas d'objet `OBSTACLE` : ça permet de "situer" l'obstacle sur le tronçon, plus ou moins (si j'ai bien compris).
Donc je vote pour garder celui-là en "obligatoire facultatif".

**Comment by Marion-Torterotot** [comment]
et je n'ai pas répondu pour `OBSTACLE.hauteurSousObs`, mais je suis d'accord pour le passer en "obligatoire facultatif" au minimum. C'est en effet une potentielle source de danger pour les personnes aveugles et malvoyantes.

**Comment by Lenoxaic** [comment]
Bonjour,
Pour gagner du temps de collecte ne garder que les obstacles pouvant engendrer un problème de sécurité ou une vraie entrave au cheminement. C'est ce qui importante au niveau des usagers. Ils ont juste besoin d'avoir l'info pour savoir s'ils vont ou pas y aller.

**Comment by GT-CNIG-DDU** [comment]
Je tente de faire la synthèse en l'état actuel de la discussion : 
- Il semble y avoir consensus pour passer `OBSTACLE.hauteurSousObs` (actuellement optionnel) en obligatoire à remplissage facultatif.
- On conserve l'attribut `OBSTACLE.longueurObstacle` (actuellement obligatoire à remplissage facultatif) notamment utile à la conversion NeTEx qui ne connaît pas la notion d'`OBSTACLE` mais la projette sur le tronçon concerné en exploitant cette longueur lorsqu'elle est renseignée.
- supprimer ou conserver `OBSTACLE.largeurObstacle` ? comme dit au début de cette discussion, il n'apparaît en effet pas gênant de le conserver car il s'agit d'un attribut optionnel, et l'éventuelle demande de collecte par la collectivité, ainsi que son chiffrage, relève moins du standard que de la négociation entre la collectivité et le prestataire de collecte. Par contre il est important que toutes les collectivités qui engagent la collecte de la largeur de l'obstacle la fasse de la même manière.

En marge de cette discussion, il a été acté ([GT CNIG Accessibilité du 24/10/2024](https://cnig.gouv.fr/IMG/pdf/241021_cr_gt_cnig_accessibilite_241010.pdf) suite à [issue 44](https://github.com/cnigfr/schema-accessibilite-voirie/issues/44)) que la classe OBSTACLE porte l'attribut obligatoire `urlMedia` à remplissage facultatif.

**Comment by Marion-Torterotot** [comment]
La synthèse me semble bien.
Je redis ici que, personnellement, je suis pour supprimer `OBSTACLE.largeurobstacle` car on peut déduire cette valeur d'après d'autres champs qui sont, eux, obligatoires. C'est un exercice mental pas si compliqué je pense, et ça peut éviter d'avoir à le collecter manuellement (ce qui rajoute tout de même bcp de mesures sur terrain...).

**Comment by GT-CNIG-DDU** [comment]
Je suis d'accord avec @Marion-Torterotot et @antoine-tesson : la largeur de l'obstacle s'avère moins utile à mesurer que `CIRCULATION.largeurUtile` et `OSTACLE.largeurUtile`.

En reformulant la synthèse : 
- passer `OBSTACLE.hauteurSousObs` (actuellement optionnel) en obligatoire à remplissage facultatif ;
- conserver l'attribut `OBSTACLE.longueurObstacle` (actuellement obligatoire à remplissage facultatif) notamment utile à la conversion NeTEx qui ne connaît pas la notion d'`OBSTACLE` mais la projette sur le tronçon concerné en exploitant cette longueur lorsqu'elle est renseignée ;
- supprimer l'attribut optionnel `OBSTACLE.largeurObstacle` qui ajoute un coût de collecte sans apporter d'information réellement utile au cheminement lorsque OSTACLE.largeurUtile et CIRCULATION.largeurUtile (tous deux à remplissage obligatoire) sont bien mesurés.

**Comment by AMIAUD1** [comment]
Pour pouvoir analyser les gênes il est nécessaire d’avoir les mesures lité à : 
OBSTACLE.largeurUtile
OBSTACLE.hauteurSousObs
Et en effet l’attribut OBSTACLE.longueurObstacle est à conserver pour NeTEx
l’attribut OBSTACLE.largeurObstacle n’est jamais déterminant dans une analyse. Il pourrait être supprimé.

**Comment by GT-CNIG-DDU** [comment]
Il y a donc consensus sur la synthèse exprimée ci-dessus. Merci @AMIAUD1.

**Comment by GT-CNIG-DDU** [comment]
Le [GT CNIG Accessibilité du 17/01/2025](https://cnig.gouv.fr/IMG/pdf/250130_cr_gt_cnig_accessibilite_250117.pdf) a acté  : 
- l'attribut `OBSTACLE.hauteurSousObs` (actuellement optionnel) devient obligatoire à remplissage facultatif ;
- l'attribut `OBSTACLE.longueurObstacle` (actuellement obligatoire à remplissage facultatif) est conservé car notamment utile à la conversion NeTEx qui ne connaît pas la notion d'OBSTACLE mais la projette sur le tronçon concerné en exploitant cette longueur lorsqu'elle est renseignée ;
- la suppression de l'attribut optionnel `OBSTACLE.largeurObstacle` car il ajoute un coût de collecte sans apporter d'information réellement utile au cheminement lorsque `OSTACLE.largeurUtile` et `CIRCULATION.largeurUtile` (tous deux à remplissage obligatoire) sont bien renseignés.

**Comment by GT-CNIG-DDU** [comment]
L'évolution a été intégrée dans le [projet de standard](https://github.com/cnigfr/schema-accessibilite-voirie/tree/main/Standard) CNIG Accessibilité.
